core/unit_test/TestReducers.hpp

```diff
@@ -1009,22 +1228,39 @@ struct TestReducers {
     test_sum(10001);
     test_prod(sizeof(Scalar) > 4 ? 35 : 19);  // avoid int overflow (see above)
     test_min(10003);
-#if !defined(KOKKOS_ENABLE_OPENACC)
-    // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
+#if !defined(KOKKOS_ENABLE_OPENACC)  // FIXME_OPENACC - OpenACC (V3.3) does not
+                                     // support custom reductions.
     test_minloc(10003);
+#if !defined(KOKKOS_COMPILER_NVHPC)       // FIXME_NVHPC misaligned memory
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET)  // FIXME_OPENMPTARGET requires custom
+                                          // reductions.
+    test_minloc_2d(100);
+#endif
+#endif
 #endif
     test_max(10007);
 #if !defined(KOKKOS_ENABLE_OPENACC)
     // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
     test_maxloc(10007);
+#if !defined(KOKKOS_COMPILER_NVHPC)       // FIXME_NVHPC misaligned memory
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET)  // FIXME_OPENMPTARGET requires custom
+                                          // reductions.
+    test_maxloc_2d(100);
+#endif
+#endif
+#endif
+#if !defined(KOKKOS_ENABLE_OPENACC)  // FIXME_OPENACC - OpenACC (V3.3) does not
+                                     // support custom reductions.
+#if defined( \
+    KOKKOS_ENABLE_OPENMPTARGET)  // FIXME_OPENMPTARGET - The minmaxloc test
+                                 // fails llvm <= 13 version, the minmaxloc_2d
+                                 // test requires custom reductions.
+#if defined(KOKKOS_COMPILER_CLANG) && (KOKKOS_COMPILER_CLANG > 1300)
```

Comment:
Not readable.  Also presumably can be enabled on the host-side and should only guarded for execution space that do not support it.